packages/discord.js/src/managers/AutoModerationRuleManager.js

```diff
@@ -57,7 +57,9 @@ class AutoModerationRuleManager extends CachedManager {
    * @property {AutoModerationRuleKeywordPresetType[]} [presets]
    * The internally pre-defined wordsets which will be searched for in the content
    * @property {string[]} [allowList] The substrings that will be exempt from triggering
-   * {@link AutoModerationRuleTriggerType.Keyword} and {@link AutoModerationRuleTriggerType.KeywordPreset}
+   * {@link AutoModerationRuleTriggerType.Keyword},
+   * {@link AutoModerationRuleTriggerType.KeywordPreset},
+   * and {@link AutoModerationRuleTriggerType.MemberProfile}
```

Comment:
```suggestion
   * {@link AutoModerationRuleTriggerType.KeywordPreset}
   * and {@link AutoModerationRuleTriggerType.MemberProfile}
   ```
   
   apples, and bananas

Reply:
discord.js prefers the use of the Oxford comma.